.github/workflows/ci.yaml

```
@@ -0,0 +1,68 @@
+# This workflow will install Python dependencies, run tests and lint with a matrix of OS and Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+  pull_request:
+    branches: [ master ]
```

It looks like all jobs ran twice, once due to the "push" event and one due to the "pull_request" event. Should this be…?

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

---

It actually replicates the current travis behaviour, tho I agree it might be redundant.
I'm also trying to cover commits to other branches as it's a nice feedback, while it's WIP.
But the proposed approach I see a problem that it'll skip other branchs.
I think it might be
```on:
  push:
    branches:
      - '!master'
  pull_request:
    branches: [ master ]
```
But i'll check it